app/src/main/res/layout/filter.xml

```diff
@@ -47,7 +47,7 @@
         <Button
             android:id="@+id/reset_button"
             android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            android:layout_height="60dp"
```

Comment:
hard-coded colors and dimensions in this file

Reply:
will do